src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/RequireThisCheck.java

```diff
@@ -437,6 +437,7 @@ private static boolean isAnonymousClassDef(DetailAST ast) {
      * or null otherwise.
      * @param ast IDENT ast to check.
      * @return the class frame where violation is found or null otherwise.
+     * @noinspection IfStatementWithIdenticalBranches
```

Comment:
@romani @Vladlis Is this why the wrecker is failing ? If yes, should I then resort to using 
 
     else {
            if(...) {
            }
    }

instead of the `else if` I've used at the line 479 ? 
Thanks